.travis.yml

```diff
@@ -15,9 +15,6 @@ php:
   - hhvm
 
 matrix:
-  include:
-    - php: 5.3
-      env: deps=low
```

Comment:
@xabbuh Ok, thank you! I found that the second test of this suite is skipped when the ``TokenStorageInterface`` is not found, so I did the same for the first test.